### PR TITLE
GitHubCI: stop restricting build+test jobs to release/next branch

### PR DESCRIPTION
**Pull request by knocte**
It is better to let contributors to FAKE run their proposed changes in GitHubCI before they propose a PR. (Without this, they wouldn't know if their changes break tests until they propose a PR.)

### CODE REVIEW

**Review by h4k1m13or — APPROVED** [review]

**Comment by mclark1129** [comment]
Just a note here, you can certainly run the workflows on your own fork if needed.  That was how I managed to get around waiting on an approval for every change I made.  https://github.com/mclark1129/FAKE/pull/3

**Comment by knocte** [comment]
@mclark1129 the branch you used for your PR was called `release/next`, that's why the workflows ran in your fork.

But the typical thing a contributor does after forking a project is creating a branch with a name that represents the thing they will work on, not having to stick to the same branch name as the original project (this way they can work on different things too and open more than 1 PR).

@yazeedobaid ping?

**Comment by knocte** [comment]
> Could you please rebase your branch from release/next? 

Done.

**Review by yazeedobaid — APPROVED** [review]
Thanks for the PR

**Pull request merged**
